### PR TITLE
Removed dependency, optimized dist size

### PR DESCRIPTION
**Pull request by Solant**
#23

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,10 +16,9 @@
     }
   ],
   "devDependencies": {
-    "browserify": "^12.0.1",
```
Comment (austinpray) [inline package.json:19]: :+1: for babel instead of browserify

File: .babelrc [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "plugins": ["transform-object-assign", "transform-es2015-modules-umd"],
```
Comment (austinpray) [inline .babelrc:2]: :+1:

File: index.js [hunk 3]
```diff
@@ -41,7 +39,7 @@ module.exports = function(el, onOver, onOut) {
 
   // Public methods
   h.options = function(opt) {
-    options = extend({}, options, opt);
+    options = Object.assign({}, options, opt);
```
Comment (austinpray) [inline index.js:42]: :+1:

**Review by austinpray — APPROVED** [review]
Good stuff

**Comment by tristen** [comment]
Thanks for your patience. Changes look good. I'll get a release out shortly 👍.

**Pull request merged**